### ISSUE TITLE
Versioning inconsistency

### ISSUE THREAD

**Issue by protist** [stephaneguindon/phyml#31]
According to [tags](https://github.com/stephaneguindon/phyml/releases), the latest stable release is 3.2.0. However, after installing, the version reported is different.
```
$ phyml-mpi --version
. command-line: phyml-mpi --version 
. This is PhyML version 20160529.
```
For package management and users knowing if they were up-to-date, I was wondering if these two could be synchronised, please?

**Comment by protist** [comment]
I just upgraded to `v3.2.20160529` according to the tags. However, the command-line version is slightly different.
```
$ phyml-mpi --version

. command-line: phyml-mpi --version 

. This is PhyML version 3.2.20160530.
```

**Comment by stephaneguindon** [comment]
I'll take a closer look at all this within the next couple of days.
Regards,
-Stephane-

**Comment by stephaneguindon** [comment]
I've posted a new release. This has tag 3.2.20160530, which should be consistent with what you get from phyml --version. The versionning in PhyML has been a real mess in the past. I intend to stick to the format used here, i.e., majorversion.minorversion.date, from now on. 
Hope that make sense,
-Stephane-

**Comment by protist** [comment]
Thank you for that! However, after installing the latest release (tagged 3.2.20160530), the command line version is still off by one.
```
$ phyml-mpi --version

. command-line: phyml-mpi --version 

. This is PhyML version 3.2.20160531.
```
I built this at local time 31 May, so I'm not sure if that is relevant.

**Comment by averagehat** [comment]
Is there some way to find the numeric version number of an older version?
i.e., I have a version from this commit: bdd18a04e820d35cfdda823f5f817d0509290acc
I know the "date" version, but need the numeric version for a publication.

```
$ phyml --version

                                                                         │
. This is PhyML version 20151007.
```

**Comment by protist** [comment]
@averagehat I investigated a bit further, also in relation to my (unanswered) comment above. It seems that the version supplied by `$ phyml --version` is just the date of the build, so for a publication, it's probably not very informative. I can't really help you with the version of the commit, but if it's untagged, you'd probably just have to reference the commit string anyway?

@stephaneguindon I rebuilt and installed the latest version, and now it's showing a different internal version, which is derived from today's date.
```
$ phyml-mpi --version

. command-line: phyml-mpi --version 

. This is PhyML version 3.2.20160622.
```
